packages/runtime/src/remote/index.ts

```diff
@@ -140,7 +140,7 @@ export class RemoteHandler {
     loadEntry: new AsyncHook<
       [
         {
-          createScriptHook: FederationHost['loaderHook']['lifecycle']['createScript'];
+          loaderHook: FederationHost['loaderHook'];
```

Comment:
break change?

Reply:
Yes, we shouldn't be able to change the name and usage

Reply:
Many users use this hook to change how scripts are loaded